### ISSUE TITLE
Starter Block freezes game

### ISSUE THREAD

**Issue by BigBadKangaroo** [tsunamayo/Starship-EVO#4887]
Hi there,

Everytime I place a starter block and try to interact with it to spawn a blueprint, the game freezes for a couple of seconds!

https://user-images.githubusercontent.com/81495956/187027206-6cc14b98-1a47-479c-8efa-327c10f7a061.mp4

**Comment by ProPeach** [comment]
I experience this too, for me it's due to having hundreds of blueprints to load up. Do you still see the same freeze if you move most of your blueprints to a different folder so the game doesn't have to show them all?

**Comment by tsunamayo** [comment]
Yes it is because of the sheer quantity of blueprint you have. I will try to improve it a bit, thanks...
